docs/setup/settings.asciidoc

```diff
@@ -306,10 +306,10 @@ the `polling` method could be used enabling that option. *Default: `false`*
 suppress all logging output. *Default: `false`*
 
 | `logging.timezone`
- | Set to the canonical time zone ID
+ | Optional. Set to the canonical time zone ID
```

Comment:
The legacy logging timezone default was [changed](https://github.com/elastic/kibana/pull/22696/files/d5035d0c96f9b4dbf1aaa46507a557d2d54c72d5#diff-346031b8ac33192f2ab14d07875be8d3c9eb6a0c5c78cee592b42841b9238f90) from UTC to the host timezone in https://github.com/elastic/kibana/pull/22696. It appears that the documentation wasn't updated to reflect the change. As it stands right now, `logging.timezone` is actually [optional](https://github.com/elastic/kibana/blob/master/packages/kbn-legacy-logging/src/log_format.ts#L92) (internally falling back to the local timezone of the host) and we needed to [update the docs](https://github.com/elastic/kibana/blob/master/docs/setup/settings.asciidoc) to reflect that.

Reply:
Update: I removed `Optional` and replaced `*Default: `UTC`* with "When not set, log events use the host timezone"